pkgs/applications/networking/instant-messengers/turses/default.nix

```diff
@@ -27,6 +34,9 @@ buildPythonPackage rec {
   '';
 
   patchPhase = ''
+    for p in $patches; do
+        patch -p1 < $p
+    done
```

Comment:
Instead of reimplementing the default `patchPhase`, you can move the  `sed` calls to `postPatch`.